src/lem/circuit.rs

```diff
@@ -1485,12 +1485,19 @@ impl Func {
             }
         }
         let globals = &mut HashSet::default();
+        let bit_decomp_cost = match F::FIELD {
+            LanguageField::Pallas => 298,
+            LanguageField::Vesta => 301,
+            LanguageField::BLS12_381 => 388,
```

Comment:
Why is this still a thing if we remove BLS12-381. If there are leftover bits that are not here just for testing, could we list them in an issue @arthurpaulino ?

Reply:
I'm not fully aware of all the places BLS12-381 can be found. This one is easy because `LanguageField` is defined in `lurk::field`, which is pretty central